src/main/java/com/yelp/nrtsearch/server/luceneserver/search/query/multifunction/MultiFunctionScoreQuery.java

```diff
@@ -252,8 +265,13 @@ public Scorer scorer(LeafReaderContext context) throws IOException {
           docSets[i] = new Bits.MatchAllBits(context.reader().maxDoc());
         }
       }
-      return new MultiFunctionScorer(
-          innerScorer, this, scoreMode, boostMode, leafFunctions, docSets);
+
+      Scorer scorer =
+          new MultiFunctionScorer(innerScorer, this, scoreMode, boostMode, leafFunctions, docSets);
+      if (minScore > 0 || minExcluded) {
```

Comment:
What if you want a minScore of 0 inclusive? It seems like you will need to differentiate the states of 0 and unset. A `google.protobuf.FloatValue` type would work for this.

Reply:
I saw lucene ticket to enforce the score to be positive in 8.8.
LUCENE-7996

In nrtsearch, I tested and if the script returns a negative number, the final score in response is 0